src/java.base/share/classes/java/util/IdentityHashMap.java

```diff
@@ -555,25 +523,14 @@ public V remove(Object key) {
     private boolean removeMapping(Object key, Object value) {
```

Comment:
Note that `IdentityHashMap.remove(Object, Object)` needs to call this method, as the default implementation calls `Objects.equals(…)`: <https://github.com/openjdk/jdk/blob/3f923b82c31325504430b50dee262fd460004e7b/src/java.base/share/classes/java/util/Map.java#L866-L874>

Reply:
This is being addressed in <https://github.com/openjdk/jdk/pull/8259>.